padertorch/contrib/examples/source_separation/tasnet/train.py

```diff
@@ -316,8 +331,8 @@ def prepare_and_train(_run, _log, trainer, train_dataset, validate_dataset,
 
     # Perform a test run to check if everything works
     trainer.test_run(
-        prepare_iterable_captured(db, train_dataset),
-        prepare_iterable_captured(db, validate_dataset),
+        prepare_dataset_captured(db, train_dataset, shuffle=True),
+        prepare_dataset_captured(db, validate_dataset, shuffle=False),
```

Comment:
I was thinking whether one should deactivate the segmenter during validation

Reply:
It's now deactivated during validation.